### ISSUE TITLE
Properly manage the seed in an uniform way for all models

### ISSUE THREAD

**Issue by gsolard** [OSS-Pole-Emploi/gabarit#151]
### Problem

In issue #60, one of the test is non determinist. One way to ensure always the same result is to be able to fix all the random seeds (random, sklearn, tensorflow, torch, huggingface). 
Having a unified seed system would permit to be able to easily reproduce past results when using models

## Concerned template

  - [x] NLP template
  - [x] NUM template
  - [x] VISION template
  - [ ] API template
  - [ ] How templates are generated - Jinja

### Solution

- **MVP:** : Be able to fix the random seed when instantiating a model

- **Possible additions:** : Explore the way the seed should be implemented to have a unified way of fixing the seed (or not).
